Formula/docx2txt.rb

```diff
@@ -0,0 +1,19 @@
+class Docx2txt < Formula
+  desc "Converts Microsoft Office docx documents to equivalent text documents"
+  homepage "http://docx2txt.sourceforge.net/"
+  url "http://downloads.sourceforge.net/docx2txt/docx2txt-1.4.tgz"
+  sha256 "b297752910a404c1435e703d5aedb4571222bd759fa316c86ad8c8bbe58c6d1b"
+
+  depends_on "perl"
+  depends_on "unzip"
+  depends_on "bash" => :optional
+
+  def install
+    system "make", "install", "CONFIGDIR=/usr/local/etc"
+  end
+
+  test do
+    system "wget", "http://calibre-ebook.com/downloads/demos/demo.docx"
```

Comment:
This should be a vendored `resource` to enable checksumming; see [`aften.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/aften.rb) for an example.